TUT/STA130F24_TUT02_Sep13.ipynb

```diff
@@ -107,53 +78,150 @@
+    "\n",
+    "    1. Stock Investment Strategy: Students are investors trying to maximize their returns in the stock market. They must decide between two investment strategies: \"diversified portfolio\" or \"focused portfolio.\" Each strategy has different probabilities of success based on market conditions.\n",
+    "        1. Diversified Portfolio: Spread investments across multiple industries.\n",
+    "        2. Focused Portfolio: Concentrate investments in a few high-potential stocks.\n",
+    "    2. Healthcare Treatment Decision: Students are healthcare professionals deciding between two treatment options for a patient's condition. Each treatment has different success rates and potential side effects.\n",
+    "        1. Treatment A: High success rate but moderate side effects.\n",
+    "        2. Treatment B: Lower success rate but minimal side effects.\n",
+    "    3. Sports Team Strategy: Students are coaches of a sports team planning their game strategy. They must decide between two tactics: \"offensive strategy\" or \"defensive strategy.\" Each strategy has different probabilities of winning based on the opponent's strengths and weaknesses.\n",
+    "        1. Offensive Strategy: Focus on scoring goals/points aggressively.\n",
+    "        2. Defensive Strategy: Prioritize defense to prevent the opponent from scoring.\n",
+    "    4. Career Path Decision: Students are recent graduates deciding between two career paths: \"corporate job\" or \"entrepreneurship.\" Each path has different probabilities of success and factors to consider, such as job security, income potential, and work-life balance.\n",
+    "        1. Corporate Job: Stable income but limited growth opportunities.\n",
+    "        2. Entrepreneurship: Higher potential for success but greater risk and uncertainty.\n",
+    "    5. Environmental Conservation Strategy: Students are environmental activists advocating for conservation efforts in a wildlife reserve. They must decide between two conservation strategies: \"habitat preservation\" or \"species reintroduction.\" Each strategy has different probabilities of achieving long-term sustainability for the ecosystem.\n",
+    "        1. Habitat Preservation: Protect existing habitats from human encroachment.\n",
+    "        2. Species Reintroduction: Reintroduce endangered species to restore ecological balance.\n",
+    "       \n",
+    "2. **[20 - 25 minutes]** Each group prepares a brief summary, approximately 3 minutes, outlining their decision and the rationale behind it. They should address the expected outcomes, the risks involved, and why they believe their choice is the best given the uncertainty. After each presentation, allow a 1-2 minutes Q&A session where other students or TAs can ask questions or challenge the decision."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "b1985697e415697b"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Homework [0 minutes]\n",
```

Comment:
Moved from above.
Edric noted (just like for above) his additions in his response comment.